personal-tuya-devices/models/TS0601/_TZE200_zl1kmjqx.yaml

```diff
@@ -0,0 +1,21 @@
+# Temperature & Humidity Sensor with LCD
+deviceLabel: LCD T+H Sensor
+profiles:
+  - normal_temphumibatt_v1
+datapoints:
+  # https://github.com/Koenkk/zigbee-herdsman-converters/blob/077d83a3d4f02c97ca7043af6be5422c987a0498/devices/tuya.js#L1300-L1304
```

Comment:
Hi.  The rate limits are correct - humidity didn't work in the existing temphumibatt driver because the readings were discarded as out-of-range.  I think you've already changed the links but let me know if you still need me to amend them.